src/app/app.component.html

```diff
@@ -1,10 +1,11 @@
-<div *ngFor="let product of products">
-  <h1>{{product.name}}</h1>
-  <h1>{{product.price}}</h1>
-  <h1>{{product.description}}</h1>
-  <img src={{product.image}} />
-  <h1>{{product.type}}</h1>
-  <h1>{{product.status}}</h1>
+<div class="container">
```

Comment:
I think you should take a look at using the ng-container here

Reply:
But, but, but I liiike the Bootstrap containers! 😄

Reply:
hmm yeah but I think you could do <ng-container class:container></ng-container> for data-binding the html but maybe thats out of scope for basic styling ;)